app/tests/spec/models/reliers/oauth.js

```diff
@@ -556,8 +556,11 @@ define(function (require, exports, module) {
       });
 
       it('throws if no scopes', (done) => {
+        relier._config.scopedKeysEnabled = true;
+        relier.set('keysJwk', 'jwk');
+
         try {
-          relier._validateKeyScope();
+          relier.wantsKeys();
```

Comment:
nit: it may be worth an explicit `assert.fail()` after calling `wantsKeys()` here; otherwise a call to `wantsKeys()` that doesn't throw, will only be detected by a failure to call `done()` and an eventual test timeout.

Alternately, chai appears to have an `assert.throws` that could be used here: http://chaijs.com/api/assert/#method_throws